CHANGELOG.md

```diff
@@ -35,6 +36,7 @@ NOTE: [`epaint`](epaint/CHANGELOG.md), [`eframe`](eframe/CHANGELOG.md), [`egui_w
 
 ### Fixed 🐛
 * Context menu now respects the theme ([#1043](https://github.com/emilk/egui/pull/1043))
+* Plot `Orientation` was not public, although fields using this type were ([#1130](https://github.com/emilk/egui/pull/1130))
```

Comment:
I wonder why the compiler doesn't stop us from doing that O.o

Reply:
There's a basic check, but it breaks down as soon as you re-export symbols.

So, `pub` doesn't mean "visible outside the crate", but rather "visible outside the crate, if the parent modules allow it" 🙂